plugins/module_utils/network/eos/config/lag_interfaces/lag_interfaces.py

```diff
@@ -253,7 +253,13 @@ def set_config(want, have):
                     ),
                 ]
             )
-
+    fallback_settings = to_set.get("fallback", {})
+    if fallback_settings:
+        commands.extend(["interface %s" % want["name"]])
```

Comment:
The top level 'interface <name>' command has to be taken outside of members and fallback_settings , in order to avoid duplicate commands.